src/homeork_2/Model/CustomArrayList.java

```diff
@@ -0,0 +1,69 @@
+package com.second.Model;
+
+import java.util.Arrays;
+
+public class CustomArrayList {
+    private static final int DEFAULT_CAPACITY = 10;
+    private Student[] students;
+    private int size;
+
+
+    public CustomArrayList() {
+        students = new Student[DEFAULT_CAPACITY];
+        size = 0;
+    }
+
+
+    public Student add(Student student) {
+        if (size == students.length) {
+            ensureCapacity();
+        }
+        students[size++] = student;
+        return student;
+    }
+
+
+    private void ensureCapacity() {
+        int newCapacity = students.length * 2;
+        students = Arrays.copyOf(students, newCapacity);
+    }
+
+
+    public int size() {
+        return size;
+    }
+
+
+    public Student get(int index) {
+        if (index < 0 || index >= size()) {
+            throw new ArrayIndexOutOfBoundsException();
+        }
+        return students[index];
+    }
+    public boolean remove(Student student) {
+        final Student[] st = students;
+        final int size = this.size;
+        int i = 0;
+        found: {
+            if (student == null) {
+                for (; i < size; i++)
```

Comment:
I think you can keep only one for instead of two
 as you didn' override equals method of Student class , the default behavior(inherited from Object class) is check by reference so you can have only  
for (; i < size; i++) {
                    if (student == st[i])
                        break found;
}